metals/src/main/scala/scala/meta/internal/builds/BloopInstallResult.scala

```diff
@@ -1,30 +0,0 @@
-package scala.meta.internal.builds
-
-import scala.meta.internal.builds.Digest.Status
-
-sealed abstract class BloopInstallResult extends Product with Serializable {
```

Comment:
This is maybe a bit opinionated, but I wanted to be able to re-use this for the reload result for build tools that support workspace reload from BSP. Since that's part of the spec, I decided to rename this `WorkspaceReloadResult` to better reflect it. As you'll see in the `WorkspaceReloadResult.scala` it's mentioned that it's used for both.